react-app/public/locales/pl/translation.json

```
@@ -46,6 +46,7 @@
   "successfulLogout": "Pomyślnie wylogowano",
   "successfulDeleteAccount": "Pomyślnie usunięto konto",
   "videosLoaded": "Znaleziono {{loadedVideosAmount}} wideo podczas przeszukiwania podanych słów kluczowych. Proszę sprawdzić we wtyczce.",
-  "fallbackError": "Wystąpił niespodziewany błąd, skontaktuj sie z developerem",
+  "fallbackError": "Wystąpił niespodziewany błąd, skontaktuj sie z developerem!",
+  "heading": "BŁĄD",
```

```suggestion
  "fatalErrorHeading": "BŁĄD",
```

cause IMO, it's more meaningful, if you have many props, then this 'heading' could be everything or nothing :)